news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,212 @@
+
+His efforts have helped make the tournament more accessible to so many people, and we are hugely thankful for his efforts.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Sartan.jpg)](https://osu.ppy.sh/users/4100941)
+
+**Sartan** has been something of a devilish carry-cross-hypeman fusion in the osu!catch World Cup tournament scene since 2017. Together with another recent contributor-nominee JBHyperion, the two together essentially *were* the commentary team for several major tournaments. His infectious positivity and playful commentary earned him the adoration of the crowd, and his incredible efforts to breathe life and hype into his favorite gamemode have earned him this acknowledgement of contribution well and truly above the norm.
+
+## Moderation
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Kobold84.jpg)](https://osu.ppy.sh/users/3227533)
+
+Holding the line in one of the game's most unruly language-based channels, **Kobold84** is the undisputed supermoderator of the Russian-speaking community, investing thousands of hours into keeping an entire subsection of the game orderly, fair and fun for everyone.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Death.jpg)](https://osu.ppy.sh/users/3242450)
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Dntm8kmeeatu.jpg)](https://osu.ppy.sh/users/5428812)
+
+Combined, **Death and Dntm8kmeeatu** put the support in the Support Team, collectively responsible for solving literally thousands of problems and answering questions for those in need, all while balancing their duties as members of the Global Moderation Team.
+
+## Development
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Milan.jpg)](https://osu.ppy.sh/users/1052994)
+
+A chief contributor to the Mappers' Guild and Beatmap Nominator infrastructure, **Milan-**'s advice and aid have helped shape the backbone of the systems that govern osu!'s ranking cycle, and one of its primary projects for promoting the use of officially licensed music, all while maintaining his place in the Beatmap Nominators.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Joehu.jpg)](https://osu.ppy.sh/users/8549835)
+
+**Joehu** has been a part of the osu!wiki team and a regular contributor to the osu!lazer project for many years now, and has rapidly made a name for himself as one of the most fastidious and efficient reviewers overall. Wikifriends near and far regularly comment that if you get a green light from Joehu, you know you're set, and when folklore legends are arising out of someone's attention to detail, there's really not much story to tell at that point.
+
+## Nominators
```

Comment:
I get mixed feelings about this cuz there's already rewards of the same type for dedicated nominators, and according to these descriptions, the basis for "elite nominator" seems to be more=better for most of the people. I think the subjective nature of what makes someone a "good" nominator will be problematic for how these badges are viewed. I feel like this is the kind of thing that should have been discussed beyond just the NAT beforehand, I doubt I'm the only person with this concern and now it's too late to make changes (I assume)

to be clear I'm mostly complaining about how this is a whole separate badge with different implications than "community contributor", which would imo be more fitting.

Reply:
they're chosen based on a review of metrics + performance as discussed with the NAT, so they're essentially contributor-like badges that have a baseline requirement for consideration

Reply:
which is why I feel like community contributor would've been more appropriate. "elite nominator" by itself gives some impression that they do *better* work than normal BNs, something you could easily disagree with if your mapping/modding opinions don't align